<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayOption</class>
 <widget class="QDockWidget" name="DisplayOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>155</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="Window title of display options like .">Display</string>
  </property>
  <widget class="QWidget" name="innerWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>576</width>
         <height>136</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="mirrorButton">
          <property name="toolTip">
           <string>Horizontal flip</string>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/app/icons/mirror.png</normaloff>:/app/icons/mirror.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::DelayedPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mirrorVButton">
          <property name="toolTip">
           <string>Vertical flip</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/app/icons/mirrorV.png</normaloff>:/app/icons/mirrorV.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="thinLinesButton">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="acceptDrops">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Show invisible lines</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/app/icons/thinlines5.png</normaloff>:/app/icons/thinlines5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="outLinesButton">
          <property name="toolTip">
           <string>Show outlines only</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/app/icons/outlines5.png</normaloff>:/app/icons/outlines5.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayCenterButton">
          <property name="toolTip">
           <string>Overlay shows field center</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayCenter.png</normaloff>:/icons/overlayCenter.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayGoldenRatioButton">
          <property name="toolTip">
           <string>Overlay shows field in Golden Ratio</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayGoldenRatio.png</normaloff>:/icons/overlayGoldenRatio.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayThirdsButton">
          <property name="toolTip">
           <string>Overlay shows field in thirds</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayThirds.png</normaloff>:/icons/overlayThirds.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlaySafeAreaButton">
          <property name="toolTip">
           <string>Overlay shows field safe areas</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlaySafe.png</normaloff>:/icons/overlaySafe.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayPerspective1Button">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayPerspective1.png</normaloff>:/icons/overlayPerspective1.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayPerspective2Button">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayPerspective2.png</normaloff>:/icons/overlayPerspective2.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayPerspective3Button">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayPerspective3.png</normaloff>:/icons/overlayPerspective3.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="overlayChooseAngle">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../data/app.qrc">
            <normaloff>:/icons/overlayAngle.png</normaloff>:/icons/overlayAngle.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="action5_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>5 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 5 degrees</string>
   </property>
  </action>
  <action name="action10_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 10 degrees</string>
   </property>
  </action>
  <action name="action15_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>15 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 15 degrees</string>
   </property>
  </action>
  <action name="action30_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>30 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 30 degrees</string>
   </property>
  </action>
  <action name="action3_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>3 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 3 degrees</string>
   </property>
  </action>
  <action name="action7_5_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>7.5 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 7.5 degrees</string>
   </property>
  </action>
  <action name="action2_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>2 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 2 degrees</string>
   </property>
  </action>
  <action name="action20_degrees">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20 degrees</string>
   </property>
   <property name="toolTip">
    <string>Angles = 20 degrees</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../data/app.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>onionPrevButtonClicked(bool)</slot>
  <slot>onionNextButtonClicked(bool)</slot>
  <slot>onionBlueButtonClicked(bool)</slot>
  <slot>onionRedButtonClicked(bool)</slot>
  <slot>toggleMirror(bool)</slot>
  <slot>toggleMirrorV(bool)</slot>
 </slots>
</ui>
